src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -71,7 +71,7 @@ static License GetTrialLicense()
             }
             else
             {
-                var message = $"Trial for Particular Service Platform is still active, trial expires on {trialLicense.ExpirationDate.Value.ToLocalTime().ToShortDateString()}.";
+                var message = $"Trial for the Particular Service Platform has been active since {trialStartDate.ToLocalTime().ToShortDateString()}.";
```

Comment:
That is _really_ verbose for a log message. And I don't want to lead with anything that gives a feeling of impending doom, because human behavior is to stop reading at that point, no matter the mitigating description afterward.